packages/harmony/src/components/layout/Paper/Paper.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from '@emotion/styled'
+
+import { toCSSVariableName } from 'utils/styles'
+
+import type { PaperProps } from './types'
+
+/** Base layout component used as a building block for creating pages and other components. */
```

Comment:
Should we be using multiline comment not part 80? I think there is some eslint thing that can possibly help